oscps-lib/src/component.rs

```diff
@@ -30,7 +30,7 @@ pub enum ChemicalIdentifier {
 impl Chemical {
 
     /// constructor
-    pub async fn new(identifier : ChemicalIdentifier) -> Result<Self> {
+    pub fn new(identifier : ChemicalIdentifier) -> Result<Self> {
```

Comment:
Good idea, asynchronous code can often lead to difficult-to-catch bugs. We can focus on async once we have an established working code.